joern-cli/frontends/javasrc2cpg/src/main/scala/io/joern/javasrc2cpg/JavaSrc2Cpg.scala

```diff
@@ -30,6 +30,7 @@ class JavaSrc2Cpg extends X2CpgFrontend[Config] {
       new MetaDataPass(cpg, language, config.inputPath).createAndApply()
       val astCreationPass = new AstCreationPass(config, cpg)
       astCreationPass.createAndApply()
+      astCreationPass.sourceParser.cleanupDelombokOutput()
```

Comment:
I tried `delombokTempDir.deleteOnExit` first, but for some reason that didn't work. Probably some unclosed `File/Reader/Writer`, but I couldn't find it.